bootstrap.py

```diff
@@ -201,6 +107,13 @@
     print("*. Skipping GUI toolkit detection")
     os.environ['QT_API'] = args.gui
 
+# ---- Update PyLSP
+
+if REPOS['python-lsp-server']['editable']:
+    # Installed in editable mode -> reinstall
+    install_repo('python-lsp-server', editable=True)
+
```

Comment:
```suggestion
```

See [comment](https://github.com/spyder-ide/spyder/pull/17408#discussion_r853526131) on the previous thread.

Reply:
Yes, I'll remove this.